doc/sphinxman/source/psithoninput.rst

```diff
@@ -216,6 +220,18 @@ the input file, so if the last four commands in the above example were to read :
 the commands that set the print level would be ineffective, as they would be
 processed after the CCSD computation completes.
 
+In PsiAPI mode::
```

Comment:
Might want to stress that things like `e_convergence` should *not* be strings. This  does run, but something very odd happens and the options do not appear to change.